triton-vm/src/fri.rs

```diff
@@ -562,8 +546,7 @@ impl<H: AlgebraicHasher> Fri<H> {
         // it is important to modify the sponge state the same way.
         prover.proof_stream.sample_scalars(1);
 
-        let indices = prover.all_top_level_collinearity_check_indices();
-        Ok(indices)
+        Ok(prover.first_round_collinearity_check_indices.clone())
```

Comment:
```suggestion
        Ok(prover.first_round_collinearity_check_indices)
```

nit: `prover` is consumed here, making the `.clone()` superfluous.